src/app/component/ProductInformation/ProductInformation.component.js

```diff
@@ -87,6 +87,29 @@ export class ProductInformation extends PureComponent {
         );
     }
 
+    isHTMLWhiteSpaceOrUndefined(htmlString) {
+        if (!htmlString || htmlString.trim() === '') {
+            return true;
+        }
+
+        const parser = new DOMParser();
+        const document = parser.parseFromString(htmlString.trim(), 'text/html');
+
+        if (document.body.children.length === 0) {
+            return false;
+        }
```

Comment:
Why ignore invalid HTML here? Let's stick to the behavior we see in browser and allow writing plain text.
![image](https://user-images.githubusercontent.com/46347627/99967136-16795600-2da0-11eb-9d70-b1da59b07c29.png)

Reply:
This condition ensures plain text is not ignored. 
*the function is supposed to return `false` if the string has content and will **not** render white space.